frontend/src/views/PasswordFind.vue

```diff
@@ -59,10 +64,20 @@ export default {
     validName: false,
     validEmail: false,
     validBirth: false,
+    showLoading: false,
   }),
   methods: {
-    submit() {
-      //TODO 비밀번호 찾기 API
+    async submit() {
+      this.showLoading = true
+      await Api.fetchPasswordFind({
+        name: this.name,
+        email: this.email,
+        birthday: DateUtil.formatLocalDate(this.birth),
+      }).catch(e => {
+        this.showLoading = false
+        throw e
+      })
+      this.$router.push("/find/result")
```

Comment:
```suggestion
      try {
        await Api.fetchPasswordFind({~~})
        this.$router.push("/find/result")
      } catch(e) {
        this.showLoading = false
        alert(e.response.message)
      }
```
이렇게 하면 분기를 위해 예외를 던질 필요도 없고, 여기서 처리할 수도 있을 것 같아요!

Reply:
아 그때 핀의 리뷰 내용이네요
저번것이랑 같이 수정해야겠어요